build/Extensions/DepsJsonExtensions.cs

```diff
@@ -125,7 +125,7 @@ public static void RollFrameworkForward(this JsonObject depsJson, string runtime
 
     public static void RemoveDuplicatedLibraries(this JsonObject depsJson, ReadOnlyCollection<AbsolutePath> architectureStores)
```

Comment:
Technically whole functionality can be removed for now, but I was introducing in twice so I would keep it doing noting.